[
{
  "model": "admin.logentry",
  "pk": 1,
  "fields": {
    "action_time": "2023-01-13T01:26:25.717Z",
    "user": 1,
    "content_type": 8,
    "object_id": "1",
    "object_repr": "What is Lorem Ipsum?",
    "action_flag": 1,
    "change_message": "[{\"added\": {}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 2,
  "fields": {
    "action_time": "2023-01-13T01:35:43.470Z",
    "user": 1,
    "content_type": 8,
    "object_id": "1",
    "object_repr": "What is Lorem Ipsum?",
    "action_flag": 2,
    "change_message": "[]"
  }
},
{
  "model": "admin.logentry",
  "pk": 3,
  "fields": {
    "action_time": "2023-01-13T01:36:08.879Z",
    "user": 1,
    "content_type": 8,
    "object_id": "1",
    "object_repr": "What is Lorem Ipsum?",
    "action_flag": 2,
    "change_message": "[]"
  }
},
{
  "model": "admin.logentry",
  "pk": 4,
  "fields": {
    "action_time": "2023-01-14T02:20:44.518Z",
    "user": 1,
    "content_type": 8,
    "object_id": "1",
    "object_repr": "What is Lorem Ipsum?",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Status\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 5,
  "fields": {
    "action_time": "2023-01-14T02:21:21.097Z",
    "user": 1,
    "content_type": 8,
    "object_id": "3",
    "object_repr": "Another new  post",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Status\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 6,
  "fields": {
    "action_time": "2023-01-14T17:18:26.003Z",
    "user": 1,
    "content_type": 8,
    "object_id": "4",
    "object_repr": "The Godfather",
    "action_flag": 1,
    "change_message": "[{\"added\": {}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 7,
  "fields": {
    "action_time": "2023-01-14T17:58:59.443Z",
    "user": 1,
    "content_type": 8,
    "object_id": "5",
    "object_repr": "Schindler's List",
    "action_flag": 1,
    "change_message": "[{\"added\": {}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 8,
  "fields": {
    "action_time": "2023-01-14T22:36:22.379Z",
    "user": 1,
    "content_type": 9,
    "object_id": "2",
    "object_repr": "Comment by First School on Schindler's List",
    "action_flag": 3,
    "change_message": ""
  }
},
{
  "model": "admin.logentry",
  "pk": 9,
  "fields": {
    "action_time": "2023-01-14T22:36:22.618Z",
    "user": 1,
    "content_type": 9,
    "object_id": "3",
    "object_repr": "Comment by First School on Schindler's List",
    "action_flag": 3,
    "change_message": ""
  }
},
{
  "model": "admin.logentry",
  "pk": 10,
  "fields": {
    "action_time": "2023-01-14T22:41:20.339Z",
    "user": 1,
    "content_type": 9,
    "object_id": "1",
    "object_repr": "Comment by First School on Schindler's List",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Active\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 11,
  "fields": {
    "action_time": "2023-01-14T23:32:54.414Z",
    "user": 1,
    "content_type": 8,
    "object_id": "5",
    "object_repr": "Schindler's List",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Tags\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 12,
  "fields": {
    "action_time": "2023-01-15T00:05:10.893Z",
    "user": 1,
    "content_type": 8,
    "object_id": "4",
    "object_repr": "The Godfather",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Tags\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 13,
  "fields": {
    "action_time": "2023-01-15T00:55:30.514Z",
    "user": 1,
    "content_type": 8,
    "object_id": "6",
    "object_repr": "Markdown-Post",
    "action_flag": 1,
    "change_message": "[{\"added\": {}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 14,
  "fields": {
    "action_time": "2023-01-15T01:01:38.678Z",
    "user": 1,
    "content_type": 8,
    "object_id": "6",
    "object_repr": "Markdown-Post",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Body\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 15,
  "fields": {
    "action_time": "2023-01-15T01:04:06.882Z",
    "user": 1,
    "content_type": 8,
    "object_id": "6",
    "object_repr": "Markdown-Post",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Body\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 16,
  "fields": {
    "action_time": "2023-01-15T01:05:44.360Z",
    "user": 1,
    "content_type": 8,
    "object_id": "6",
    "object_repr": "Markdown-Post",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Body\"]}}]"
  }
},
{
  "model": "admin.logentry",
  "pk": 17,
  "fields": {
    "action_time": "2023-01-15T01:19:03.211Z",
    "user": 1,
    "content_type": 12,
    "object_id": "1",
    "object_repr": "localhost:8000",
    "action_flag": 2,
    "change_message": "[{\"changed\": {\"fields\": [\"Domain name\", \"Display name\"]}}]"
  }
},
{
  "model": "auth.permission",
  "pk": 1,
  "fields": {
    "name": "Can add log entry",
    "content_type": 1,
    "codename": "add_logentry"
  }
},
{
  "model": "auth.permission",
  "pk": 2,
  "fields": {
    "name": "Can change log entry",
    "content_type": 1,
    "codename": "change_logentry"
  }
},
{
  "model": "auth.permission",
  "pk": 3,
  "fields": {
    "name": "Can delete log entry",
    "content_type": 1,
    "codename": "delete_logentry"
  }
},
{
  "model": "auth.permission",
  "pk": 4,
  "fields": {
    "name": "Can view log entry",
    "content_type": 1,
    "codename": "view_logentry"
  }
},
{
  "model": "auth.permission",
  "pk": 5,
  "fields": {
    "name": "Can add permission",
    "content_type": 2,
    "codename": "add_permission"
  }
},
{
  "model": "auth.permission",
  "pk": 6,
  "fields": {
    "name": "Can change permission",
    "content_type": 2,
    "codename": "change_permission"
  }
},
{
  "model": "auth.permission",
  "pk": 7,
  "fields": {
    "name": "Can delete permission",
    "content_type": 2,
    "codename": "delete_permission"
  }
},
{
  "model": "auth.permission",
  "pk": 8,
  "fields": {
    "name": "Can view permission",
    "content_type": 2,
    "codename": "view_permission"
  }
},
{
  "model": "auth.permission",
  "pk": 9,
  "fields": {
    "name": "Can add group",
    "content_type": 3,
    "codename": "add_group"
  }
},
{
  "model": "auth.permission",
  "pk": 10,
  "fields": {
    "name": "Can change group",
    "content_type": 3,
    "codename": "change_group"
  }
},
{
  "model": "auth.permission",
  "pk": 11,
  "fields": {
    "name": "Can delete group",
    "content_type": 3,
    "codename": "delete_group"
  }
},
{
  "model": "auth.permission",
  "pk": 12,
  "fields": {
    "name": "Can view group",
    "content_type": 3,
    "codename": "view_group"
  }
},
{
  "model": "auth.permission",
  "pk": 13,
  "fields": {
    "name": "Can add user",
    "content_type": 4,
    "codename": "add_user"
  }
},
{
  "model": "auth.permission",
  "pk": 14,
  "fields": {
    "name": "Can change user",
    "content_type": 4,
    "codename": "change_user"
  }
},
{
  "model": "auth.permission",
  "pk": 15,
  "fields": {
    "name": "Can delete user",
    "content_type": 4,
    "codename": "delete_user"
  }
},
{
  "model": "auth.permission",
  "pk": 16,
  "fields": {
    "name": "Can view user",
    "content_type": 4,
    "codename": "view_user"
  }
},
{
  "model": "auth.permission",
  "pk": 17,
  "fields": {
    "name": "Can add content type",
    "content_type": 5,
    "codename": "add_contenttype"
  }
},
{
  "model": "auth.permission",
  "pk": 18,
  "fields": {
    "name": "Can change content type",
    "content_type": 5,
    "codename": "change_contenttype"
  }
},
{
  "model": "auth.permission",
  "pk": 19,
  "fields": {
    "name": "Can delete content type",
    "content_type": 5,
    "codename": "delete_contenttype"
  }
},
{
  "model": "auth.permission",
  "pk": 20,
  "fields": {
    "name": "Can view content type",
    "content_type": 5,
    "codename": "view_contenttype"
  }
},
{
  "model": "auth.permission",
  "pk": 21,
  "fields": {
    "name": "Can add session",
    "content_type": 6,
    "codename": "add_session"
  }
},
{
  "model": "auth.permission",
  "pk": 22,
  "fields": {
    "name": "Can change session",
    "content_type": 6,
    "codename": "change_session"
  }
},
{
  "model": "auth.permission",
  "pk": 23,
  "fields": {
    "name": "Can delete session",
    "content_type": 6,
    "codename": "delete_session"
  }
},
{
  "model": "auth.permission",
  "pk": 24,
  "fields": {
    "name": "Can view session",
    "content_type": 6,
    "codename": "view_session"
  }
},
{
  "model": "auth.permission",
  "pk": 25,
  "fields": {
    "name": "Can add Logging",
    "content_type": 7,
    "codename": "add_statuslog"
  }
},
{
  "model": "auth.permission",
  "pk": 26,
  "fields": {
    "name": "Can change Logging",
    "content_type": 7,
    "codename": "change_statuslog"
  }
},
{
  "model": "auth.permission",
  "pk": 27,
  "fields": {
    "name": "Can delete Logging",
    "content_type": 7,
    "codename": "delete_statuslog"
  }
},
{
  "model": "auth.permission",
  "pk": 28,
  "fields": {
    "name": "Can view Logging",
    "content_type": 7,
    "codename": "view_statuslog"
  }
},
{
  "model": "auth.permission",
  "pk": 29,
  "fields": {
    "name": "Can add post",
    "content_type": 8,
    "codename": "add_post"
  }
},
{
  "model": "auth.permission",
  "pk": 30,
  "fields": {
    "name": "Can change post",
    "content_type": 8,
    "codename": "change_post"
  }
},
{
  "model": "auth.permission",
  "pk": 31,
  "fields": {
    "name": "Can delete post",
    "content_type": 8,
    "codename": "delete_post"
  }
},
{
  "model": "auth.permission",
  "pk": 32,
  "fields": {
    "name": "Can view post",
    "content_type": 8,
    "codename": "view_post"
  }
},
{
  "model": "auth.permission",
  "pk": 33,
  "fields": {
    "name": "Can add comment",
    "content_type": 9,
    "codename": "add_comment"
  }
},
{
  "model": "auth.permission",
  "pk": 34,
  "fields": {
    "name": "Can change comment",
    "content_type": 9,
    "codename": "change_comment"
  }
},
{
  "model": "auth.permission",
  "pk": 35,
  "fields": {
    "name": "Can delete comment",
    "content_type": 9,
    "codename": "delete_comment"
  }
},
{
  "model": "auth.permission",
  "pk": 36,
  "fields": {
    "name": "Can view comment",
    "content_type": 9,
    "codename": "view_comment"
  }
},
{
  "model": "auth.permission",
  "pk": 37,
  "fields": {
    "name": "Can add tag",
    "content_type": 10,
    "codename": "add_tag"
  }
},
{
  "model": "auth.permission",
  "pk": 38,
  "fields": {
    "name": "Can change tag",
    "content_type": 10,
    "codename": "change_tag"
  }
},
{
  "model": "auth.permission",
  "pk": 39,
  "fields": {
    "name": "Can delete tag",
    "content_type": 10,
    "codename": "delete_tag"
  }
},
{
  "model": "auth.permission",
  "pk": 40,
  "fields": {
    "name": "Can view tag",
    "content_type": 10,
    "codename": "view_tag"
  }
},
{
  "model": "auth.permission",
  "pk": 41,
  "fields": {
    "name": "Can add tagged item",
    "content_type": 11,
    "codename": "add_taggeditem"
  }
},
{
  "model": "auth.permission",
  "pk": 42,
  "fields": {
    "name": "Can change tagged item",
    "content_type": 11,
    "codename": "change_taggeditem"
  }
},
{
  "model": "auth.permission",
  "pk": 43,
  "fields": {
    "name": "Can delete tagged item",
    "content_type": 11,
    "codename": "delete_taggeditem"
  }
},
{
  "model": "auth.permission",
  "pk": 44,
  "fields": {
    "name": "Can view tagged item",
    "content_type": 11,
    "codename": "view_taggeditem"
  }
},
{
  "model": "auth.permission",
  "pk": 45,
  "fields": {
    "name": "Can add site",
    "content_type": 12,
    "codename": "add_site"
  }
},
{
  "model": "auth.permission",
  "pk": 46,
  "fields": {
    "name": "Can change site",
    "content_type": 12,
    "codename": "change_site"
  }
},
{
  "model": "auth.permission",
  "pk": 47,
  "fields": {
    "name": "Can delete site",
    "content_type": 12,
    "codename": "delete_site"
  }
},
{
  "model": "auth.permission",
  "pk": 48,
  "fields": {
    "name": "Can view site",
    "content_type": 12,
    "codename": "view_site"
  }
},
{
  "model": "auth.user",
  "pk": 1,
  "fields": {
    "password": "pbkdf2_sha256$390000$2kHgxpJMmhbiNrV9ueNXSu$UtnYvJB4AFmbEXUJyilHRMU2ApKso2y55hKmOqTAvkA=",
    "last_login": "2023-01-13T01:22:18.958Z",
    "is_superuser": true,
    "username": "Admin",
    "first_name": "",
    "last_name": "",
    "email": "",
    "is_staff": true,
    "is_active": true,
    "date_joined": "2023-01-13T01:21:05.436Z",
    "groups": [],
    "user_permissions": []
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 1,
  "fields": {
    "app_label": "admin",
    "model": "logentry"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 2,
  "fields": {
    "app_label": "auth",
    "model": "permission"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 3,
  "fields": {
    "app_label": "auth",
    "model": "group"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 4,
  "fields": {
    "app_label": "auth",
    "model": "user"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 5,
  "fields": {
    "app_label": "contenttypes",
    "model": "contenttype"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 6,
  "fields": {
    "app_label": "sessions",
    "model": "session"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 7,
  "fields": {
    "app_label": "django_db_logger",
    "model": "statuslog"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 8,
  "fields": {
    "app_label": "blog",
    "model": "post"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 9,
  "fields": {
    "app_label": "blog",
    "model": "comment"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 10,
  "fields": {
    "app_label": "taggit",
    "model": "tag"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 11,
  "fields": {
    "app_label": "taggit",
    "model": "taggeditem"
  }
},
{
  "model": "contenttypes.contenttype",
  "pk": 12,
  "fields": {
    "app_label": "sites",
    "model": "site"
  }
},
{
  "model": "sessions.session",
  "pk": "1gis4a125vapzf6mfz7orc7jx25vuw4f",
  "fields": {
    "session_data": ".eJxVjEEOwiAQRe_C2pAyFDAu3XsGMswMUjWQlHZlvLsh6UK3_7333yrivpW4d1njwuqijDr9bgnpKXUAfmC9N02tbuuS9FD0Qbu-NZbX9XD_Dgr2MuqcTDAhB5cpSLaTBEuzZCAHHhwKMUzWUxKPzM6bGZjxnCDYbNmC-nwBCIo4tw:1pG8lv:FlJdvmFxRDuMB-syEIF-qUAsOUABEdkpaXSOxhK0A94",
    "expire_date": "2023-01-27T01:22:19.029Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 1,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 8, in post_list\n    return render(request,'blog/post/list.xhtml',{'posts': posts})\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/list.xhtml",
    "create_datetime": "2023-01-14T01:52:35.763Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 2,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 8, in post_list\n    return render(request,'blog/post/list.xhtml',{'posts': posts})\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/list.xhtml",
    "create_datetime": "2023-01-14T02:06:59.682Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 3,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/14/the-godfather/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\nTypeError: post_detail() got an unexpected keyword argument 'year'",
    "create_datetime": "2023-01-14T17:35:32.520Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 4,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/14/the-godfather/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/fields/__init__.py\", line 2018, in get_prep_value\n    return int(value)\nValueError: invalid literal for int() with base 10: 'day'\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 12, in post_detail\n    post = get_object_or_404(Post,status=Post.Status.PUBLISHED,slug=post,publish__year=year, publish__month='month', publish__day='day')\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 85, in get_object_or_404\n    return queryset.get(*args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 636, in get\n    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1421, in filter\n    return self._filter_or_exclude(False, args, kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1439, in _filter_or_exclude\n    clone._filter_or_exclude_inplace(negate, args, kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1446, in _filter_or_exclude_inplace\n    self._query.add_q(Q(*args, **kwargs))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1532, in add_q\n    clause, _ = self._add_q(q_object, self.used_aliases)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1562, in _add_q\n    child_clause, needed_inner = self.build_filter(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1478, in build_filter\n    condition = self.build_lookup(lookups, col, value)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1303, in build_lookup\n    lookup = lookup_class(lhs, rhs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 27, in __init__\n    self.rhs = self.get_prep_lookup()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 341, in get_prep_lookup\n    return super().get_prep_lookup()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 85, in get_prep_lookup\n    return self.lhs.output_field.get_prep_value(self.rhs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/fields/__init__.py\", line 2020, in get_prep_value\n    raise e.__class__(\nValueError: Field 'None' expected a number but got 'day'.",
    "create_datetime": "2023-01-14T17:37:33.653Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 5,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/14/the-godfather/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/fields/__init__.py\", line 2018, in get_prep_value\n    return int(value)\nValueError: invalid literal for int() with base 10: 'day'\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 12, in post_detail\n    post = get_object_or_404(Post,status=Post.Status.PUBLISHED,slug=post,publish__year=year, publish__month='month', publish__day='day')\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 85, in get_object_or_404\n    return queryset.get(*args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 636, in get\n    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1421, in filter\n    return self._filter_or_exclude(False, args, kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1439, in _filter_or_exclude\n    clone._filter_or_exclude_inplace(negate, args, kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/query.py\", line 1446, in _filter_or_exclude_inplace\n    self._query.add_q(Q(*args, **kwargs))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1532, in add_q\n    clause, _ = self._add_q(q_object, self.used_aliases)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1562, in _add_q\n    child_clause, needed_inner = self.build_filter(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1478, in build_filter\n    condition = self.build_lookup(lookups, col, value)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/sql/query.py\", line 1303, in build_lookup\n    lookup = lookup_class(lhs, rhs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 27, in __init__\n    self.rhs = self.get_prep_lookup()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 341, in get_prep_lookup\n    return super().get_prep_lookup()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/lookups.py\", line 85, in get_prep_lookup\n    return self.lhs.output_field.get_prep_value(self.rhs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/db/models/fields/__init__.py\", line 2020, in get_prep_value\n    raise e.__class__(\nValueError: Field 'None' expected a number but got 'day'.",
    "create_datetime": "2023-01-14T17:37:45.005Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 6,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 13, in post_list\n    posts = paginator.page(page_number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 75, in page\n    number = self.validate_number(number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 53, in validate_number\n    if number > self.num_pages:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 102, in num_pages\n    if self.count == 0 and not self.allow_empty_first_page:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 97, in count\n    return len(self.object_list)\nTypeError: object of type 'function' has no len()",
    "create_datetime": "2023-01-14T17:51:13.049Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 7,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 13, in post_list\n    posts = paginator.page(page_number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 75, in page\n    number = self.validate_number(number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 53, in validate_number\n    if number > self.num_pages:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 102, in num_pages\n    if self.count == 0 and not self.allow_empty_first_page:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 97, in count\n    return len(self.object_list)\nTypeError: object of type 'function' has no len()",
    "create_datetime": "2023-01-14T17:56:14.231Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 8,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 13, in post_list\n    posts = paginator.page(page_number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 75, in page\n    number = self.validate_number(number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 53, in validate_number\n    if number > self.num_pages:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 102, in num_pages\n    if self.count == 0 and not self.allow_empty_first_page:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 97, in count\n    return len(self.object_list)\nTypeError: object of type 'function' has no len()",
    "create_datetime": "2023-01-14T17:59:34.689Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 9,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 13, in post_list\n    posts = paginator.page(page_number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 75, in page\n    number = self.validate_number(number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 53, in validate_number\n    if number > self.num_pages:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 102, in num_pages\n    if self.count == 0 and not self.allow_empty_first_page:\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/utils/functional.py\", line 57, in __get__\n    res = instance.__dict__[self.name] = self.func(instance)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 97, in count\n    return len(self.object_list)\nTypeError: object of type 'function' has no len()",
    "create_datetime": "2023-01-14T18:02:50.016Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 10,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 13, in post_list\n    posts = paginator.page(page_number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 75, in page\n    number = self.validate_number(number)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/paginator.py\", line 57, in validate_number\n    raise EmptyPage(_(\"That page contains no results\"))\ndjango.core.paginator.EmptyPage: That page contains no results",
    "create_datetime": "2023-01-14T18:08:04.156Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 11,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 220, in _get_response\n    response = response.render()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/response.py\", line 114, in render\n    self.content = self.rendered_content\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/response.py\", line 90, in rendered_content\n    template = self.resolve_template(self.template_name)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/response.py\", line 72, in resolve_template\n    return select_template(template, using=self.using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 47, in select_template\n    raise TemplateDoesNotExist(\", \".join(template_name_list), chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/list.html, blog/post_list.html",
    "create_datetime": "2023-01-14T18:39:32.106Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 12,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/4/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 204, in _get_response\n    self.check_response(response, callback)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 332, in check_response\n    raise ValueError(\nValueError: The view blog.views.post_share didn't return an HttpResponse object. It returned None instead.",
    "create_datetime": "2023-01-14T21:24:42.096Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 13,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/4/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 204, in _get_response\n    self.check_response(response, callback)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 332, in check_response\n    raise ValueError(\nValueError: The view blog.views.post_share didn't return an HttpResponse object. It returned None instead.",
    "create_datetime": "2023-01-14T21:27:30.996Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 14,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/4/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:28:55.311Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 15,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/4/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:29:51.504Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 16,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/5/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:29:57.900Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 17,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/5/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:30:57.396Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 18,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/5/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:31:08.985Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 19,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/3/share/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 75, in post_share\n    return render(request, 'blog/post/share.html', {'post': post,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T21:31:18.678Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 20,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/14/the-godfather/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 57, in post_detail\n    return render(request,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/detail.html",
    "create_datetime": "2023-01-14T22:26:06.357Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 21,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/14/the-godfather/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 57, in post_detail\n    return render(request,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/detail.html",
    "create_datetime": "2023-01-14T22:26:47.423Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 22,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/5/comment/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/views/decorators/http.py\", line 43, in inner\n    return func(request, *args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 113, in post_comment\n    return render(request, 'blog/post/comment.html',\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/comment.html",
    "create_datetime": "2023-01-14T22:35:05.890Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 23,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/5/comment/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/views/decorators/http.py\", line 43, in inner\n    return func(request, *args, **kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 113, in post_comment\n    return render(request, 'blog/post/comment.xhtml',\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/base.html",
    "create_datetime": "2023-01-14T22:35:34.966Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 24,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/13/first-post/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/defaulttags.py\", line 1027, in find_library\n    return parser.libraries[name]\nKeyError: 'blog_tags'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 72, in post_detail\n    return render(request,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 132, in render\n    compiled_parent = self.get_parent(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 129, in get_parent\n    return self.find_template(parent, context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 107, in find_template\n    template, origin = context.template.engine.find_template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 157, in find_template\n    template = loader.get_template(name, skip=skip)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loaders/cached.py\", line 57, in get_template\n    template = super().get_template(template_name, skip)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loaders/base.py\", line 28, in get_template\n    return Template(\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 154, in __init__\n    self.nodelist = self.compile_nodelist()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 200, in compile_nodelist\n    return parser.parse()\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 513, in parse\n    raise self.error(token, e)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 511, in parse\n    compiled_result = compile_func(self, token)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/defaulttags.py\", line 1089, in load\n    lib = find_library(parser, name)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/defaulttags.py\", line 1029, in find_library\n    raise TemplateSyntaxError(\ndjango.template.exceptions.TemplateSyntaxError: 'blog_tags' is not a registered tag library. Must be one of:\nadmin_list\nadmin_modify\nadmin_urls\ncache\ni18n\nl10n\nlog\nstatic\ntz",
    "create_datetime": "2023-01-15T00:13:53.774Z"
  }
},
{
  "model": "django_db_logger.statuslog",
  "pk": 25,
  "fields": {
    "logger_name": "django.request",
    "level": 40,
    "msg": "Internal Server Error: /blog/2023/1/13/first-post/",
    "trace": "Traceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 62, in render\n    return self.template.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 175, in render\n    return self._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader_tags.py\", line 157, in render\n    return compiled_parent._render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 167, in _render\n    return self.nodelist.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in render\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 1005, in <listcomp>\n    return SafeString(\"\".join([node.render_annotated(context) for node in self]))\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/base.py\", line 966, in render_annotated\n    return self.render(context)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/library.py\", line 269, in render\n    t = context.template.engine.get_template(self.filename)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 175, in get_template\n    template, origin = self.find_template(template_name)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/engine.py\", line 161, in find_template\n    raise TemplateDoesNotExist(name, tried=tried)\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/latest_posts.html\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/exception.py\", line 55, in inner\n    response = get_response(request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/core/handlers/base.py\", line 197, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/django4byexample/Blog_two/blog/views.py\", line 72, in post_detail\n    return render(request,\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/shortcuts.py\", line 24, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/loader.py\", line 62, in render_to_string\n    return template.render(context, request)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 64, in render\n    reraise(exc, self.backend)\n  File \"/home/threat_hunter/Desktop/DJ_FOR_THE_IMPATIENT/dj_for_impatient/lib/python3.10/site-packages/django/template/backends/django.py\", line 85, in reraise\n    raise new from exc\ndjango.template.exceptions.TemplateDoesNotExist: blog/post/latest_posts.html",
    "create_datetime": "2023-01-15T00:22:18.926Z"
  }
},
{
  "model": "blog.post",
  "pk": 1,
  "fields": {
    "title": "What is Lorem Ipsum?",
    "slug": "first-post",
    "author": 1,
    "body": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
    "publish": "2023-01-13T01:25:21Z",
    "created": "2023-01-13T01:26:25.716Z",
    "updated": "2023-01-14T02:20:44.516Z",
    "status": "PB"
  }
},
{
  "model": "blog.post",
  "pk": 3,
  "fields": {
    "title": "Another new  post",
    "slug": "another-new-post",
    "author": 1,
    "body": "new post body",
    "publish": "2023-01-14T00:45:45Z",
    "created": "2023-01-14T00:45:45.838Z",
    "updated": "2023-01-14T02:21:21.095Z",
    "status": "PB"
  }
},
{
  "model": "blog.post",
  "pk": 4,
  "fields": {
    "title": "The Godfather",
    "slug": "the-godfather",
    "author": 1,
    "body": "The purpose of lorem ipsum is to create a natural looking block of text (sentence, paragraph, page, etc.) that doesn't distract from the layout. A practice not without controversy, laying out pages with meaningless filler text can be very useful when the focus is meant to be on design, not content.",
    "publish": "2023-01-14T17:17:03Z",
    "created": "2023-01-14T17:18:26.002Z",
    "updated": "2023-01-15T00:05:10.887Z",
    "status": "PB"
  }
},
{
  "model": "blog.post",
  "pk": 5,
  "fields": {
    "title": "Schindler's List",
    "slug": "schindlers-list",
    "author": 1,
    "body": "Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Cicero's De Finibus Bonorum et Malorum for use in a type specimen book. It usually begins with:",
    "publish": "2023-01-14T17:58:32Z",
    "created": "2023-01-14T17:58:59.442Z",
    "updated": "2023-01-14T23:32:54.403Z",
    "status": "PB"
  }
},
{
  "model": "blog.post",
  "pk": 6,
  "fields": {
    "title": "Markdown-Post",
    "slug": "markdown-post",
    "author": 1,
    "body": "This is a post formatted with markdown\r\n-------------------------------------------\r\n#*This is emphasized* and **This is more emphasized**\r\n\r\nHere is a list :\r\n\r\n- First item\r\n- Second item\r\n- Third item\r\n\r\nAnd a [link to Django Website](http://www.django.com).",
    "publish": "2023-01-15T00:53:03Z",
    "created": "2023-01-15T00:55:30.499Z",
    "updated": "2023-01-15T01:05:44.353Z",
    "status": "PB"
  }
},
{
  "model": "blog.comment",
  "pk": 1,
  "fields": {
    "post": 5,
    "name": "First School",
    "email": "test@example.com",
    "body": "aasa",
    "created": "2023-01-14T22:35:05.759Z",
    "updated": "2023-01-14T22:41:20.338Z",
    "active": false
  }
},
{
  "model": "blog.comment",
  "pk": 4,
  "fields": {
    "post": 5,
    "name": "Jose",
    "email": "john@gmail.com",
    "body": "greta",
    "created": "2023-01-14T22:40:57.108Z",
    "updated": "2023-01-14T22:40:57.108Z",
    "active": true
  }
},
{
  "model": "taggit.tag",
  "pk": 1,
  "fields": {
    "name": "django",
    "slug": "django"
  }
},
{
  "model": "taggit.tag",
  "pk": 2,
  "fields": {
    "name": "jazz",
    "slug": "jazz"
  }
},
{
  "model": "taggit.tag",
  "pk": 3,
  "fields": {
    "name": "music",
    "slug": "music"
  }
},
{
  "model": "taggit.tag",
  "pk": 4,
  "fields": {
    "name": "python",
    "slug": "python"
  }
},
{
  "model": "taggit.tag",
  "pk": 5,
  "fields": {
    "name": "markdown",
    "slug": "markdown"
  }
},
{
  "model": "taggit.tag",
  "pk": 6,
  "fields": {
    "name": "html",
    "slug": "html"
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 2,
  "fields": {
    "tag": 2,
    "content_type": 8,
    "object_id": 1
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 3,
  "fields": {
    "tag": 3,
    "content_type": 8,
    "object_id": 1
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 4,
  "fields": {
    "tag": 4,
    "content_type": 8,
    "object_id": 5
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 5,
  "fields": {
    "tag": 2,
    "content_type": 8,
    "object_id": 4
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 6,
  "fields": {
    "tag": 4,
    "content_type": 8,
    "object_id": 4
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 7,
  "fields": {
    "tag": 4,
    "content_type": 8,
    "object_id": 6
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 8,
  "fields": {
    "tag": 5,
    "content_type": 8,
    "object_id": 6
  }
},
{
  "model": "taggit.taggeditem",
  "pk": 9,
  "fields": {
    "tag": 6,
    "content_type": 8,
    "object_id": 6
  }
},
{
  "model": "sites.site",
  "pk": 1,
  "fields": {
    "domain": "localhost:8000",
    "name": "localhost:8000"
  }
}
]
